Clase 02/styles.css

```diff
@@ -0,0 +1,22 @@
+body {
+    background-color: #ac644b;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
```

Comment:
Okay, that makes sense.
But if I want to keep it scalable, idk if in the future there will be a div or another structure ( different from text ). So, wouldn't it be better to leave it as it is?

Reply:
Well that's kind of my whole point. To keep it scalable you don't want to set the entire body to flex-direction column since the default direction is always row. 
In fact you don't want to set any style to the body unless it's removing default margin, padding and setting box-sizing to border-box like we learn in this week's lesson.  
It may be a better idea to create a div that takes 100% of the with and center its content or to directly apply styles to h1 and p tags (with classes and ids). 
Also you can create a utils class called .centered-text that you can re-use every time you need a text to be centered. 
I hope this helps 🙃

Reply:
Reach me out if you have doubts! Happy to help

Reply:
Oh okay, thx for the explanation :)